### ISSUE TITLE
Add HTML as a third type of input source file

### ISSUE THREAD

**Issue by greghendershott** [greghendershott/frog#72]
# Use case and rationale

A user has a big pile of HTML files from something like Blogger or LiveJournal. The user is willing to add the standard Frog "front matter" to the files (`Title/Date/Tags`), but beyond that doesn't want to edit/change them.

They _could_ give these files a `.md` extension to be parsed as markdown, on the theory that HTML is valid markdown. However:
1. Actually the markdown spec is somewhat picky about [how/where HTML is allowed](http://daringfireball.net/projects/markdown/syntax#html). Although some markdown parsers try to be more lenient, they vary. There are tricky corner cases.
2. It would be faster to parse the HTML _as_ straight HTML, using e.g. Racket's `read-html-as-xml`.
3. It would be fastest of all not to parse the HTML at all -- just ploink it as-is in the `content` section of `page-template.html` or `post-template.html`.
# tl;dr:

Add `.html` as a third type of input source file, in addition to `.md` and `.scrbl`.

cf issue #67.
